https://git.reviewboard.kde.org/r/121589/

Index: ksnakeduel/CMakeLists.txt
===================================================================
--- ksnakeduel.orig/CMakeLists.txt	2015-03-28 12:03:08.059680706 +0100
+++ ksnakeduel/CMakeLists.txt	2015-03-28 12:03:08.055680868 +0100
@@ -1,4 +1,4 @@
-project(ktron)
+project(ksnakeduel)
 
 if(NOT COMPILING_KDEGAMES)
 	find_package(KDE4 REQUIRED)
@@ -17,7 +17,7 @@
 
 ########### next target ###############
 
-set(ktron_SRCS
+set(ksnakeduel_SRCS
 	renderer.cpp
 	object.cpp
 	snakepart.cpp
@@ -30,24 +30,24 @@
 	ktron.cpp
 	main.cpp)
 
-kde4_add_ui_files(ktron_SRCS general.ui)
+kde4_add_ui_files(ksnakeduel_SRCS general.ui)
 
-kde4_add_kcfg_files(ktron_SRCS settings.kcfgc)
+kde4_add_kcfg_files(ksnakeduel_SRCS settings.kcfgc)
 
-kde4_add_app_icon(ktron_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/pics/hi*-app-ktron.png")
+kde4_add_app_icon(ksnakeduel_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/pics/hi*-app-ksnakeduel.png")
 
-kde4_add_executable(ktron ${ktron_SRCS})
+kde4_add_executable(ksnakeduel ${ksnakeduel_SRCS})
 
-target_link_libraries(ktron kdegames kdegamesprivate ${KDE4_KDEUI_LIBS} ${KDE4_KNOTIFYCONFIG_LIBS})
+target_link_libraries(ksnakeduel kdegames kdegamesprivate ${KDE4_KDEUI_LIBS} ${KDE4_KNOTIFYCONFIG_LIBS})
 
-install(TARGETS ktron ${INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS ksnakeduel ${INSTALL_TARGETS_DEFAULT_ARGS})
 
 configure_file(kdesnake.cmake ${CMAKE_CURRENT_BINARY_DIR}/kdesnake  @ONLY)
 
 ########### install files ###############
 
 install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/kdesnake DESTINATION ${BIN_INSTALL_DIR})
-install(PROGRAMS  ktron.desktop kdesnake.desktop DESTINATION ${XDG_APPS_INSTALL_DIR})
+install(PROGRAMS  ksnakeduel.desktop kdesnake.desktop DESTINATION ${XDG_APPS_INSTALL_DIR})
 install(FILES  ktron.kcfg DESTINATION ${KCFG_INSTALL_DIR})
 install(FILES  ktronui.rc DESTINATION ${DATA_INSTALL_DIR}/ktron)
 install(FILES  ktron.knsrc DESTINATION ${CONFIG_INSTALL_DIR})
Index: ksnakeduel/kdesnake.cmake
===================================================================
--- ksnakeduel.orig/kdesnake.cmake	2015-03-28 12:03:08.059680706 +0100
+++ ksnakeduel/kdesnake.cmake	2015-03-28 12:03:08.055680868 +0100
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-#  KTron: KSnake mode start script ( @KDE_VERSION_STRING@ )
+#  KDE Snake: KSnake mode start script ( @KDE_VERSION_STRING@ )
 #
 
-ktron --snake
+kdesnake --snake
Index: ksnakeduel/ksnakeduel.desktop
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ ksnakeduel/ksnakeduel.desktop	2015-03-28 12:03:08.055680868 +0100
@@ -0,0 +1,53 @@
+[Desktop Entry]
+Type=Application
+Exec=ktron -caption %c %i
+Icon=ktron
+X-DocPath=ktron/index.html
+Terminal=false
+Name=KSnakeDuel
+Name[bs]=KSnakeDuel
+Name[ca]=KSnakeDuel
+Name[ca@valencia]=KSnakeDuel
+Name[cs]=KSnakeDuel
+Name[da]=KSnakeDuel
+Name[de]=KSnakeDuel
+Name[el]=KSnakeDuel
+Name[en_GB]=KSnakeDuel
+Name[es]=KSnakeDuel
+Name[et]=KSnakeDuel
+Name[fi]=KSnakeDuel
+Name[fr]=KSnakeDuel
+Name[ga]=KSnakeDuel
+Name[gl]=KSnakeDuel
+Name[hu]=KSnakeDuel
+Name[it]=KSnakeDuel
+Name[kk]=KSnakeDuel
+Name[km]=KSpaceDuel
+Name[ko]=KSnakeDuel
+Name[lt]=KSpaceDuel
+Name[lv]=KSpaceDuel
+Name[mr]=के-स्नेक-ड्युएल
+Name[nb]=KSnakeDuel
+Name[nds]=KSnakeDuel
+Name[nl]=KSnakeDuel
+Name[pl]=KSnakeDuel
+Name[pt]=KSnakeDuel
+Name[pt_BR]=KSnakeDuel
+Name[ro]=KSnakeDuel
+Name[ru]=Дуэль KSnake
+Name[sk]=KSnakeDuel
+Name[sl]=KSnakeDuel
+Name[sr]=К‑змијски-двобој
+Name[sr@ijekavian]=К‑змијски-двобој
+Name[sr@ijekavianlatin]=K‑zmijski-dvoboj
+Name[sr@latin]=K‑zmijski-dvoboj
+Name[sv]=Ormspel
+Name[tr]=KSnakeDuel
+Name[ug]=KSnakeDuel
+Name[uk]=KSnakeDuel
+Name[x-test]=xxKSnakeDuelxx
+Name[zh_CN]=KSnakeDuel
+Name[zh_TW]=KSnakeDuel
+X-KDE-StartupNotify=true
+X-DCOP-ServiceType=Multi
+Categories=Qt;KDE;Game;ArcadeGame;
Index: ksnakeduel/ktron.desktop
===================================================================
--- ksnakeduel.orig/ktron.desktop	2015-03-28 12:03:36.438514203 +0100
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,53 +0,0 @@
-[Desktop Entry]
-Type=Application
-Exec=ktron -caption %c %i
-Icon=ktron
-X-DocPath=ktron/index.html
-Terminal=false
-Name=KSnakeDuel
-Name[bs]=KSnakeDuel
-Name[ca]=KSnakeDuel
-Name[ca@valencia]=KSnakeDuel
-Name[cs]=KSnakeDuel
-Name[da]=KSnakeDuel
-Name[de]=KSnakeDuel
-Name[el]=KSnakeDuel
-Name[en_GB]=KSnakeDuel
-Name[es]=KSnakeDuel
-Name[et]=KSnakeDuel
-Name[fi]=KSnakeDuel
-Name[fr]=KSnakeDuel
-Name[ga]=KSnakeDuel
-Name[gl]=KSnakeDuel
-Name[hu]=KSnakeDuel
-Name[it]=KSnakeDuel
-Name[kk]=KSnakeDuel
-Name[km]=KSpaceDuel
-Name[ko]=KSnakeDuel
-Name[lt]=KSpaceDuel
-Name[lv]=KSpaceDuel
-Name[mr]=के-स्नेक-ड्युएल
-Name[nb]=KSnakeDuel
-Name[nds]=KSnakeDuel
-Name[nl]=KSnakeDuel
-Name[pl]=Pojedynek węży
-Name[pt]=KSnakeDuel
-Name[pt_BR]=KSnakeDuel
-Name[ro]=KSnakeDuel
-Name[ru]=Дуэль KSnake
-Name[sk]=KSnakeDuel
-Name[sl]=KSnakeDuel
-Name[sr]=К‑змијски-двобој
-Name[sr@ijekavian]=К‑змијски-двобој
-Name[sr@ijekavianlatin]=K‑zmijski-dvoboj
-Name[sr@latin]=K‑zmijski-dvoboj
-Name[sv]=Ormspel
-Name[tr]=KSnakeDuel
-Name[ug]=KSnakeDuel
-Name[uk]=KSnakeDuel
-Name[x-test]=xxKSnakeDuelxx
-Name[zh_CN]=KSnakeDuel
-Name[zh_TW]=KSnakeDuel
-X-KDE-StartupNotify=true
-X-DCOP-ServiceType=Multi
-Categories=Qt;KDE;Game;ArcadeGame;
